src/components/insulin/SlidingScaleForm.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { StyleSheet, View, Picker } from 'react-native';
+import { FormikValues, Formik } from 'formik';
+import {
+   DefaultTheme,
+   Button,
+   TextInput as PaperTextInput,
+} from 'react-native-paper';
+
+import { Entypo, Fontisto } from '@expo/vector-icons';
+
+// Custom components
+import TextInput from '../TextInput';
+
+//import { SlidingScaleTable } from '../../@types/insulin';
+
+import { InsulinDose } from '../../@types/insulin';
+import { Meal } from '../../@types/meals';
+import Dropdown from '../Dropdown';
+
+type InitialValues = {
+   totalInsulinUnits?: number;
+   bloodGlucose?: number;
+   doseLevel: string;
```

Comment:
Move this to the top since its the only value in the interface that *is* required, aka on top of the `totalInsulinUnits`